NightscoutServiceKit/RemoteCommands/NightscoutRemoteCommand.swift

```diff
@@ -0,0 +1,39 @@
+//
```

Comment:
This is Nightscout's implementation of the RemoteCommand protocol. This will be used for both Remote 1.0 and Remote 2.0.